src/data/markdown/docs/05 Examples/01 Examples/05 data-parameterization.md

```diff
@@ -16,14 +16,21 @@ This will, in turn, make your test more realistic.
 
 <Blockquote mod="warning">
 
-#### ⚠️ Strive to keep the data files small
-
-Each VU in k6 will have its separate copy of the data file.
-If your script uses 300 VUs, there will be 300 copies of the data file in memory.
-Cloud service allots 8GB of memory for every 300VUs.
-When executing cloud tests, make sure your data files aren't exceeding this limit or your test run may get aborted.
-
-Starting with k6 v0.27.0, there are some [tricks that can be used to better handle bigger data files](#handling-bigger-data-files).
+Each VU in k6 is a separate JS VM so in order to not have multiple copies of the whole data file
+[SharedArray](/javascript-api/k6-data/sharedarray) was added. It does have overhead in accessing elements compared to a normal non shared
+array, but the difference is negligible compared to the time it takes to make requests. And is
+especially not comparable as not using it with big files basically means k6 will be using too much
+memory to run. Which might lead to your script not being able to run at all or aborting in the
+middle.
```

Comment:
```suggestion
[SharedArray](/javascript-api/k6-data/sharedarray) was added. It does have some CPU overhead in accessing elements compared to a normal non shared
array, but the difference is negligible compared to the time it takes to make requests. This becomes
even less of an issue compared to not using it with large files, as k6 would otherwise use too much memory to run, which might lead to your script not being able to run at all or aborting in the middle if the system resources are exhausted.
```